update july - rules (#694)
akeneo/pim-helpcenter: content/updates/2020-07/2-rules-date-format.md

---
pim_announcement_audience:
- EE
---

> # A new relative date format

Reviewer: ```suggestion
# New rules condition on relative dates
```
I wonder if the expression "relative date" is understandable by users

Reviewer: Missing meta data too

> This is now possible to filter products/product models based on a **relative date** in the rules engine. This new format is available via YML import only for now, and for these 3 filters: `updated`, `created`, and `date` attribute type.

Reviewer: ```suggestion
You can now create a rule based on a **relative date** condition. This new condition can be currently used only through YML import (it will be shortly available in our brand new rules engine screen!) and for the `updated` and `created` system attributes and the `date` attribute type.
```

> It was already possible to filter products/product models on their `creation date`, `updated date`, or a `date` attribute type. But it was not enough.

Reviewer: ```suggestion
It's already possible to apply conditions on products/product models based on their `creation date`, `updated date`, or the content of a `date` attribute type. But we wanted to go further, by allowing you to not only filter on **specific dates** but also **relative dates**. With this new functionality, you'll be able to apply an action on all products created/updated since the past X days. 
```

> Indeed, these filters expected only **specific dates** as values.

Reviewer: ```suggestion
```

> For instance, you could parameter your rule in order to be able to *filter on products/product models that contains the value "06/23/20" for the "release date" attribute.*.

Reviewer: ```suggestion
```

> But in some cases, you may need to filter on **relative dates** as well. It can be very useful if you need to select all the products/product models that have been ***updated on the last 24 hours***, for example.

Reviewer: ```suggestion
```
